archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
Ok, I'll remove encoding and `type: str` then? It would be up to the application to decide how to treat these byte arrays (i.e. using current locale encoding, some in-app setting or something like that)